minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -129,3 +129,28 @@ java.arg.7=-Djava.security.egd=file:/dev/urandom
 
 #Set headless mode by default
 java.arg.14=-Djava.awt.headless=true
+
+# MiNiFi Command & Control Configuration
+# C2 Properties
+# Enabling C2 Uncomment each of the following options
+#c2.enable=true
```

Comment:
With the current implementation we also need to highlight that FileChangeIngestor needs to be enabled to make the c2 protocol based update fully work
e.g.:
```
nifi.minifi.notifier.ingestors=org.apache.nifi.minifi.bootstrap.configuration.ingestors.FileChangeIngestor
nifi.minifi.notifier.ingestors.file.config.path=./conf/config-new.yml
nifi.minifi.notifier.ingestors.file.polling.period.seconds=5
```